### PR TITLE
Made it (more) responsive

### PR DESCRIPTION
**Pull request by EdieLemoine**
Also tidied a lot of code which is why it says there are so many edits.

### CODE REVIEW

File: PokeClicker/system.css [hunk 1]
```diff
@@ -192,24 +202,24 @@ table, #console, #currentEnemy, #upgradeBox, #navBar, #townView, #gymView {
 @-moz-keyframes spinleft { 100% { -moz-transform: rotate(20deg); } }
 @-webkit-keyframes spinleft { 100% { -webkit-transform: rotate(20deg); } }
 
-    @keyframes spinleft {
```
Comment (Ishadijcks) [inline PokeClicker/system.css]: What is going on here? Why does it think it has changed?

**Comment by Ishadijcks** [comment]
Can you explain how it is more responsive now?

**Comment by EdieLemoine** [comment]
It now scales properly on any size, mobile isn't perfect as the map doesn't scale up or down but on any size I tested (between mobile and 4K) the map container fits properly and doesn't leave empty space. Also I changed some padding/margins to all be the same size.

**Comment by Ishadijcks** [comment]
Yeah map will be fixed in a future update, where I make the overflow scrollable.
Alright that's great! I really hadn't designed the interface with mobile and 4K in mind, but it's cool that that works now.

I'm gonna play with the width a little bit, because right now it's a tad too wide for my taste.

Thanks :)

**Comment by Ishadijcks** [comment]
I might have overlooked something, but where is the leftColumn getting its margin-left from?

**Comment by EdieLemoine** [comment]
What do you mean? Like how it's floating in the middle?

**Comment by Ishadijcks** [comment]
Ya

**Comment by EdieLemoine** [comment]
Because .container does that, you used .container-fluid before which is essentially a full-width container. This one floats in the middle, has a 100% width but a max-width of 1170px (or something) so it's fullwidth on anything smaller than its max-width.

**Comment by Ishadijcks** [comment]
Ooh yeah I get it.

**Pull request closed without merging**